cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.proto

```diff
@@ -41,13 +41,13 @@ service CloudProvider {
 
   // PricingNodePrice returns a theoretical minimum price of running a node for
   // a given period of time on a perfectly matching machine.
-  // Implementation optional.
+  // Implementation optional: if so should return error code 12 Unimplemented
```

Comment:
nit: the wording feels a little ambiguous. Here's a suggestion:  
```suggestion
  // Implementation optional: if unimplemented return error code 12 (for `Unimplemented`) 
```

Reply:
ok, suggestion accepted